great-idea/services.html

```diff
@@ -15,102 +16,102 @@
+    <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
   </nav>
 
-<header>
+  <header>
     <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
-</header>
+  </header>
 
-<section class="services-top-content">
+  <section class="services-top-content">
     <h1>Services</h1>
-    <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
-</section>
+    <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce
+      finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+  </section>
 
-<hr>
+  <hr>
 
-<section class="flexy-flex">
+  <section class="flexy-flex">
     <div>
       <h2>Digital Design</h2>
       <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-      eleifend. Mauris euismod facilisis iaculis.</p>
-      
+        eleifend. Mauris euismod facilisis iaculis.</p>
+
       <button>Learn More</button>
     </div>
-    
+
     <div>
-        <h2>UX / UI</h2>
-        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-            eleifend. Mauris euismod facilisis iaculis.</p>
-        
-        <button>Learn More</button>
+      <h2>UX / UI</h2>
+      <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+        eleifend. Mauris euismod facilisis iaculis.</p>
+
+      <button>Learn More</button>
     </div>
 
     <div>
       <h2>Digital Marketing</h2>
       <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-          eleifend. Mauris euismod facilisis iaculis.</p>
-      
+        eleifend. Mauris euismod facilisis iaculis.</p>
+
       <button>Learn More</button>
     </div>
 
     <div>
```

Comment:
For the divs that go with your boxes/cards, i'd like to see some class names on them. Even if you feel like you don't need to edit them, you will find it a good practice in the future to assign a class to wrapping HTML elements/tags. And since all the boxes/cards have the same styling, you could assign all 6 divs the same class name, so when you are selecting the boxes in CSS, you can kill 6 birds in one shot :)